File: Python/Seção06/Exe10.py
#entradas
idade = int(input("Informe a idade:"))
#procesamento
if idade >= 5 and idade <= 7:
    print("Infantil A")
elif idade >= 8 and idade <= 11:
    print("Infantil B")
elif idade >= 12 and idade <= 13:
    print("Juvenil A")
elif idade >= 14 and idade <= 17:
    print("Junenil B")
elif idade >= 18:
    print("Adulto")
elif idade < 5:
    print("Não poderá competir!")

File: Java/src/br/com/dominio/s06/Exe10.java
package br.com.dominio.s06;

import java.util.Scanner;

public class Exe10 {

	public static void main(String[] args) {
		// variaveis
		int idade;
		Scanner teclado = new Scanner(System.in);
		
		//entradas
		System.out.println("Informe a idade do nadador: ");
		idade = teclado.nextInt();
		
		//processamento
		if (idade >= 5 && idade <= 7){
			System.out.println("Infantil A");
		}else if(idade >= 8 && idade <= 11) {
			System.out.println("Infantil B");
		}else if(idade >= 12 && idade <= 13) {
			System.out.println("Juvenil A");
		}else if(idade >= 14 && idade <= 17) {
			System.out.println("Juvenil B");
		}else if(idade >=18) {
			System.out.println("Adulto");
		}
		teclado.close();
	}

}
